### ISSUE TITLE
Problem setting up plugin (crashing server)

### ISSUE THREAD

**Issue by ph1l** [tkurki/signalk-to-influxdb#5]
```
pi@freeboard:~/signalk-server-node $ ./bin/signalk-server -s settings/freeboard-serial.json
TypeError: Cannot read property 'length' of undefined
    at Object.start (/home/pi/signalk-to-influxdb/index.js:113:39)
    at registerPlugin (/home/pi/signalk-server-node/lib/interfaces/plugins.js:115:12)
    at fs.readdirSync.filter.forEach.e (/home/pi/signalk-server-node/lib/interfaces/plugins.js:93:7)
    at Array.forEach (native)
    at startPlugins (/home/pi/signalk-server-node/lib/interfaces/plugins.js:85:68)
    at Object.start (/home/pi/signalk-server-node/lib/interfaces/plugins.js:26:7)
    at /home/pi/signalk-server-node/lib/index.js:196:30
    at /home/pi/signalk-server-node/node_modules/lodash/lodash.js:4944:15
    at Function.forIn (/home/pi/signalk-server-node/node_modules/lodash/lodash.js:12938:11)
    at startInterfaces (/home/pi/signalk-server-node/lib/index.js:188:5)
```

**Comment by ph1l** [comment]
The plugins config:

```
pi@freeboard:~/signalk-server-node $ cat plugin-config-data/signalk-to-influxdb.json                                                                            
{
  "enabled": true,
  "configuration": {
    "host": "localhost",
    "port": 8086,
    "database": "boatdata",
    "blackOrWhite": "Black"
  },
}
```

**Comment by ph1l** [comment]
manually adding configuration.`blackOrWhitelist: []` seems to fix this..

**Comment by sbender9** [comment]
Yep, my fault. Fix coming...

**Comment by tkurki** [comment]
Fixed in #6
